enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -1349,9 +1349,14 @@
                       [{:model "Card"          :id c1-1-eid  :label "question_1_1"}]
                       ;; card that the card on dashboard linked to
                       [{:model "Card"          :id c1-2-eid  :label "question_1_2"}]}
-                 (->> (extract/extract-subtrees {:targets [["Collection" coll4-id]]})
-                      (map serdes.base/serdes-path)
-                      set)))))))))
+                    (->> (extract/extract-subtrees {:targets [["Collection" coll4-id]]})
+                         (map serdes.base/serdes-path)
+                         set)))))
+
+        (testing "data-model-only"
+          (let [results (into [] (extract/extract-metabase {:data-model-only true}))]
+            (is (seq (by-model "Table" results)))
```

Comment:
I realize this is minimal, but we can either expand on this PR for data-model-only tests or do another issue/PR for it specifically, since PR is just about keeping the functionality the same.